test/renate/shift/test_mmd_helpers.py

```diff
@@ -0,0 +1,39 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+import torch
+
+from renate.shift.mmd_helpers import RBFKernel
+
+
```

Comment:
There are no tests for `mmd` and `mmd_gram`

Reply:
They are touched via the test for the MMD detector, but I will add similar tests that directly invoke these functions.